laser_encoders/validate_models.py

```diff
@@ -0,0 +1,43 @@
+import os
+import tempfile
+
+import pytest
+
+from laser_encoders.download_models import LaserModelDownloader
+from laser_encoders.language_list import LASER2_LANGUAGE, LASER3_LANGUAGE
+from laser_encoders.laser_tokenizer import initialize_tokenizer
+from laser_encoders.models import initialize_encoder
+
+
+@pytest.mark.parametrize("lang", LASER3_LANGUAGE)
+def test_validate_language_models_and_tokenize_laser3(lang):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        print(f"Created temporary directory for {lang}", tmp_dir)
+
+        downloader = LaserModelDownloader(model_dir=tmp_dir)
+        downloader.download_laser3(lang)
```

Comment:
> Some languages do not pass the test. While I attempted to list their names, it's a time-consuming process due to the extensive validation time required by the models.

Can we monkeypatch the downloader (just for the test), so instead of downloading the model, it is retrieving the model out of some cache?

This way, we could test two things separately:
1. Whether all the languages are matched to an existing language code correctly (with a mocked downloader)?
2. Does the downloader work (with a real downloader, but with only a few languages).

Reply:
Do the models need to be downloaded into the cache beforehand for this?